### PR TITLE
update windows  path for docker

### PR DESCRIPTION
**Pull request by d4rk5h0t**
top directory c in windows needs admin privilege

### CODE REVIEW

File: docs/guide/run-ansible.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ collecting the rewards.
 ::: tab 🪟 Window
 
 ```
-docker run -i --rm -v c:\zarb\:/zarb zarb/zarb key generate -p /zarb/keystore/validator_key.json
-docker run -i --rm -v c:\zarb\:/zarb zarb/zarb key generate -p /zarb/keystore/mintbase_key.json
+docker run -i --rm -v %USERPROFILE%\zarb:/zarb zarb/zarb key generate -p /zarb/keystore/validator_key.json
+docker run -i --rm -v %USERPROFILE%\zarb:/zarb zarb/zarb key generate -p /zarb/keystore/mintbase_key.json
 ```
 
 This command will generate new keys and save them at: `c:\zarb\keystore\`
```
Comment (themantre) [inline docs/guide/run-ansible.md:71]: I am not a Windows user, but I think we can't spoon-feed them. This is an example to show how a key can be generated

Also, Look at line #71, it is still `c:\zarb\keystore\`. We have this location in other docs as well.

Comment (d4rk5h0t) [reply docs/guide/run-ansible.md]: sure but we ain't spoon-feeding them just providing working examples to try out

**Comment by b00f** [comment]
I am not sure if users want to use power shell. Even they know about it!
I believe users should find their way of deployment.

**Comment by d4rk5h0t** [comment]
> I am not a Windows user, but I think we can't spoon-feed them. This is an example to show how a key can be generated

@b00f 
sure just as 
themantre point out I'm just trying to provide working example and lessen the issues made on project for no reason

**Pull request closed without merging**